kjourney/src/app/view-tabs/view-tabs.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="container">
+  <ngb-tabset>
+
+    <ngb-tab title="Job Family Diagrams">
```

Comment:
what is a job family diagram

Reply:
A diagram with the diagrams of Job Families